### PR TITLE
feat: mailto option after form activation

### PR DESCRIPTION
**Pull request by mantariksh**
Related to #9 

## Screenshots
image

image

## Tests
- [ ] Activating storage mode form shows the Toastr as per the screenshot. Toastr lasts for 10 seconds. Clicking on the link opens an email with the form title, URL and secret key, also as per the screenshot.

### CODE REVIEW

File: src/public/modules/forms/services/mailto.client.factory.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  const generateMailToUri = (title, secretKey, formId) => {
+    // Construct body piece by piece to avoid confusion with order of operations
+    let body = dedent`
+      Dear collaborator,
+
+      I am sharing my form's secret key with you for safekeeping and backup. This is an important key that is needed to access all form responses.
+
+      Form title: ${title}`
+    if (formId) {
+      body += `Form URL: https://form.gov.sg/${formId}`
```
Comment (arshadali172) [inline src/public/modules/forms/services/mailto.client.factory.js]: we should not be hard coding in `https://form.gov.sg`. Please use the defaults available?

Comment (mantariksh) [reply src/public/modules/forms/services/mailto.client.factory.js]: agree, you mean `LINKS.APP.ROOT` in the translations right?

Comment (mantariksh) [reply src/public/modules/forms/services/mailto.client.factory.js]: that resolves to `https://form.gov.sg` anyway, but it won't be a hardcode

Comment (arshadali172) [reply src/public/modules/forms/services/mailto.client.factory.js]: `window.appUrl` i believe

File: src/public/modules/forms/services/mailto.client.factory.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+   */
+  const generateMailToUri = (title, secretKey, formId) => {
+    // Construct body piece by piece to avoid confusion with order of operations
+    let body = dedent`
+      Dear collaborator,
+
+      I am sharing my form's secret key with you for safekeeping and backup. This is an important key that is needed to access all form responses.
+
+      Form title: ${title}`
+    if (formId) {
```
Comment (arshadali172) [inline src/public/modules/forms/services/mailto.client.factory.js:22]: why would `formId` not be defined?

Comment (mantariksh) [reply src/public/modules/forms/services/mailto.client.factory.js:22]: we also use this function in the create form modal, but at that point the form has not been created yet so it doesn't have an ID, so the `formId` parameter is optional in this function

**Review by arshadali172 — APPROVED** [review]
Minor suggestions

File: src/public/modules/forms/services/mailto.client.factory.js (hunk 2 above)
Comment (arshadali172) [reply src/public/modules/forms/services/mailto.client.factory.js:22]: got it

File: src/public/modules/forms/services/mailto.client.factory.js (hunk 1 above)
Comment (arshadali172) [reply src/public/modules/forms/services/mailto.client.factory.js]: lets use the translations file instead. That seems to be what is consistently used by the frontend. so `LINKS.APP.ROOT`

**Comment by mantariksh** [comment]
@karrui requesting your review to double check that I'm using the translations correctly

**Review by karrui — APPROVED** [review]
lgtm

**Pull request merged**